### ISSUE TITLE
Translation stopping at japanese dot (。)

### ISSUE THREAD

**Issue by finia2NA** [Saravananslb/py-googletranslation#20]
**Describe the bug**
The Translation of text seems to stop at a japanese dot (。), which is used frequently to mark the end of a sentence. This is not the case on the google translate website, or when using regular dots.

**To Reproduce**
The following exerpt is from a session with python 3.8.5 and pygoogletranslation 2.0.5
```
❯ python
Python 3.8.5 (default, Jan 27 2021, 15:41:15) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.

>>> from pygoogletranslation import Translator
>>> t = Translator()

>>> t.translate("おはようございます。私の名前はカンナです。").text
'Good morning.'

>>> t.translate("おはようございます.私の名前はカンナです.").text
'Good morning. My name is Kanna.'
```

As can be seen, when using japanese dots, only the first sentence "good morning" is translated. The second sentence "My name is Kanna" is not. This is not the case when using english dots.

**Expected behavior**
The whole text should be translated, like it was when using english dots.

**Desktop (please complete the following information):**
 - OS: Manjaro Linux
 - Python: 3.8.5
 - Version 2.0.5

**Comment by finia2NA** [comment]
I have begun [working on this myself](https://github.com/finia2NA/py-googletranslation). Not entirely sure how to do this though. It seems google sends back parts that would make up the correct translation, but they are not being assembled correctly. My quick and dirty fix would be to just replace the 。 with . before sending text away.

I clarified what is needed to run and test the program, and created a Pipfile for convenience. And I deleted the __pycache__ in the pygoogletranslation folder.
